tests/arrow_tests.py

```diff
@@ -1207,6 +1207,22 @@ def test_aware_tz(self):
             self.assertEqual(f.tzinfo, tz.gettz("US/Central"))
             self.assertEqual(c.tzinfo, tz.gettz("US/Central"))
 
+    def test_bounds_param_is_passed(self):
```

Comment:
Hey @kkoziara, sorry for the delay. Quick question, the reason you're mainly testing on `span` is because both `span_range` and `interval` use `span` under the hood right?

Reply:
Yes, exactly